README.md

```diff
@@ -124,17 +123,17 @@ $ sudo apt-get install python-rosdep python-catkin-tools
 
 For Create 2 (Roomba 600/700 series):
 ``` bash
-$ roslaunch create_bringup create_2.launch
+$ ros2 launch create_bringup create_2.xml
```

Comment:
I don't think renaming the launch files is necessary. Correct me if I'm wrong.

Reply:
I tried to use `.launch`, but `ros2 launch` would try to interpret them as python ROS2 launch files. Adding the `.xml` made it correctly parse then as XML launch files, but there might be a better way of doing it -- I'm open to suggestions.